spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/resolver/AetherModuleResolver.java

```diff
@@ -169,4 +154,100 @@ public void serviceCreationFailed(Class<?> type, Class<?> impl, Throwable except
 		});
 		return locator.getService(RepositorySystem.class);
 	}
+
+
+	/**
+	 * Resolve a set of artifacts based on their coordinates, including their dependencies, and return the locations of
+	 * the transitive set in the local repository. Aether performs the normal Maven resolution process ensuring that the
+	 * latest update is cached to the local repository. A number of additional includes and excludes can be specified,
+	 * allowing to override the transitive dependencies of the original set. Includes and their transitive dependencies
+	 * will always
+	 *
+	 * @param coordinates the Maven coordinates of the artifacts
+	 * @return a {@link FileSystemResource} representing the resolved artifact in the local repository
+	 * @throws RuntimeException if the artifact does not exist or the resolution fails
+	 */
+	@Override
+	public Resource[] resolve(Coordinates[] coordinates, Coordinates[] includes, String[] excludePatterns) {
+		Assert.notEmpty(coordinates, "A list of root nodes must be provided");
+		for (Coordinates coordinate : coordinates) {
+			validateCoordinates(coordinate);
+		}
+		if (!ObjectUtils.isEmpty(includes)) {
+			for (Coordinates include : includes) {
+				validateCoordinates(include);
+			}
+		}
+		List<Resource> result = new ArrayList<>();
+		// if we only have one artifact to resolve, it will be set as root
```

Comment:
Actually, are you referring to https://github.com/spring-cloud/spring-cloud-stream/pull/132/files#diff-3ca29f22b67f5c13875e045bd003bd6fR183 ? This uses `null` as first arg and several includes, which is not the same thing. Or am I missing something ?